framework/include/utils/InputParameters.h

```diff
@@ -1303,4 +1318,10 @@ InputParameters::getParamHelper(const std::string & name,
 
 InputParameters emptyInputParameters();
 
+// This is temporary specialization to allow applications to utilize MultiMooseEnum for
```

Comment:
I can understand why you don't have deprecated warnings in here, but do you have a plan to come find all these temporary workarounds to take them out later? Perhaps a searchable tag in the source? Do you have a ticket open for this task?

Reply:
The steps will be:
1. Merge this.
2. Update apps for changes 
3. Enable these deprecated messages 
4. Remove deprecated messages after some time

I will open a ticket when I get to 3.

Reply:
I will also add a tag of some sort so I can find them all.